source/ClusterAPI/Setup.rst

```diff
@@ -55,192 +45,132 @@ cloud Ubuntu image, not the stripped down CAPI image designed for nodes.
-    clusterctl generate cluster $CLUSTER_NAME \
-    --kubernetes-version v1.x.y \
-    --control-plane-machine-count=3 \
-    --worker-machine-count=1 > $CLUSTER_NAME.yaml
+These steps only have to be completed once per management machine.
 
-- Edit the generated `$CLUSTER_NAME.yaml` file to specify the allocated floating IP. 
+Openstack Preparation
+---------------------
 
-.. warning::
+- `Ensure a dedicated floating IP exists. <https://openstack.stfc.ac.uk/project/floating_ips/>`_ If required, allocate an IP to the project from the External pool.
+- Clone https://github.com/DavidFair/scd-capi-values (this will be upstreamed soon)
 
-    If the selected floating IP is already being used by an existing load balancer in the
-    same project it will be disassociated and re-allocated to the new load balancer.
+clouds.yaml Prep
+----------------
 
-The existing block needs changing from:
+- Generate your application credentials: :ref:`Openstack Application Credentials<application_credentials>`. 
+  It is recommended you use Horizon (the web interface) to download the clouds.yaml file.
+- Move your clouds.yaml file into the `scd-capi-values` directory, it should have the following format:
 
 .. code-block:: yaml
 
-    spec:
-      apiServerLoadBalancer:
-        enabled: true
+    clouds:
+        openstack:
+            auth:
+            auth_url: https://openstack.stfc.ac.uk:5000/v3
+            application_credential_id: ""
+            application_credential_secret: ""
+        region_name: "RegionOne"
+        interface: "public"
+        identity_api_version: 3
+        auth_type: "v3applicationcredential"
 
-To the floating IP pre-allocated in the project:
+- Add the UUID of the project to create the cluster in. This can be found `here <https://openstack.stfc.ac.uk/identity/>`_.
 
-.. code-block:: yaml
+Your clouds.yaml should now look like:
 
-    spec:
-      apiServerFloatingIP: 130.246.x.y
-      apiServerLoadBalancer:
-        enabled: true
-
-This ensures the cluster load balancer will always use the same address, and will not
-use the entire project's quota allocating new floating IPs if there are any problems.
+.. code-block:: yaml
 
-Provisioning the new cluster
-----------------------------
+    clouds:
+        openstack:
+            auth:
+            auth_url: https://openstack.stfc.ac.uk:5000/v3
+            application_credential_id: ""
+            application_credential_secret: ""
+            project_id: ""
+        region_name: "RegionOne"
+        interface: "public"
+        identity_api_version: 3
+        auth_type: "v3applicationcredential"
 
-- Create the cluster by applying the generated cluster definition:
+- Place this file in the scd-capi-values directory you cloned earlier.
 
-.. code-block:: bash
+Creating the cluster
+====================
 
-    kubectl apply -f $CLUSTER_NAME.yaml
+Configuring the cluster
+-----------------------
 
```

Comment:
Maybe we could add a summary of what each YAML file is used for by the Helm chart here?